src/scripts/towns/TownList.ts

```diff
@@ -4557,8 +4557,8 @@ const PaniolaTownActor = new NPC('Actor Meredith', [
     'You want to know which one can only be found on the farm? I\'m sure you can figure that out yourself. Simple process of elimination, really.',
 ], {image: 'assets/images/trainers/Actor.png'});
 const RoyalAvenueSpectator = new NPC('Spectator', [
-    'I think battles in the Battle Royal Dome are more like games of chance. But Battle Royals are nothing compared to trying to evolve an Alolan Raichu with a Thunderstone.',
-    'Evolving Pikachu or Exeggcute in Alola can result in a new form! Sometimes.',
+    'I think battles in the Battle Royal Dome are more like games of chance. But Battle Royals are nothing compared to trying to evolve a Pikachu with a Thunderstone.',
```

Comment:
The dialogue being about "games of chance" no longer makes much sense when it's no longer talking about random evos, so might be better to rewrite it entirely rather than just tweak the old version

Reply:
Changed the dialogue accordingly and added some fun fluff about pancakes as a reference to Alolan Raichu's lore in the games